src/app/components/album/album.css

```diff
@@ -35,11 +35,15 @@ leaflet.folded .album .album__cover {
   font-size: 14px;
 }
 
+.album label.album__year {
+  font-size: 14px;
```

Comment:
We really shouldn't hardcode font-sizes like that. Not a blocker for this particular PR since it's used elsewhere, but eventually, we need to remove these.

Reply:
Agreed. GTK offers `@define-color` for reduce hard-coded colors, but it doesn't for other types. Neither `var()` seems to be unavailable for GTK so without preprocessors would that be difficult?

Reply:
rem seems to do the trick:
![image](https://user-images.githubusercontent.com/27908024/145033855-02e87396-c1fa-4ae4-b65d-d14f084b2e73.png)